jukebox/src/main/webapp/js/dbScript.js

```diff
@@ -0,0 +1,77 @@
+const NO_VIDEO = -1;
+
+var db = firebase.firestore();
+
+//Adds a song to the queue
+function addSongDB(song_ID, title, room_ID) {
+    db.collection('songs').where("SONG_ID", "==", song_ID)
+    .get()
+    .then(function(querySnapshot) {
+        if(querySnapshot.exists){ //Song is already in database, just add to queue
+            db.collection("queue").doc(song_ID).set( {
+                SONG_ID: song_ID,
+                ROOM_ID: room_ID,
+                DATE_ADDED: Date.now()
```

Comment:
Using the date of when it was added is a bit tricky - we'd be relying on the time the browser sends the database to determine what the order is. If I set my system clock to say that its July 9, 1994, my song will always come first because of it. 

What about using some sort of auto-incrementing id for each time we add a new song to the queue?